no-computer.md

```diff
@@ -11,34 +11,47 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+People are always looking for fun ways to exert dominance over their peers.  Sports, video games, and board games are all activities that can satisfy this need.  Sports are especially popular due to their added benefit of involving aerobic exercise, and two great sports to play are badminton and ping-pong.  Both are great because they are one-on-one (if you win, you are simply better than your opponent - they have no team to blame) and they involve hitting things, which further emphasizes dominance over one's opponent (it'd be even better if you could directly hit your opponent, but then they might be considered martial arts rather than sports).  However, each has its cons, as follows:
+- Ping pong requires a ping-pong table, which is expensive and takes up a lot of space.
+- In both games, unless you are an experienced player, hitting the ball/birdie with any notable amount of force will cause it to go flying out of the playing field.  This is no fun.
+- Badminton rackets look dumb.
+
+My DSL, pingminton, combines both sports to create a truly electrifying experience that lacks the downsides of either parent sport.  It takes the paddles from ping-pong and the birdie/net from badminton.  When playing pingminton, you don't need a table, you don't have to carry around dumb-looking badminton rackets, and (most importantly) you can whack the birdy with a large amount of force without it going too far.  This combination of ease-of-play, looking not lame, and getting to use lots of force creates the perfect environment in which to exert dominance over a peer.
+
```

Comment:
This sounds like an infomercial where someone is playing badminton in black and white and looks super lame, and then all of a sudden  gets a ping pong paddle and looks awesome.